genai-perf/tests/test_perf_analyzer_config.py

```diff
@@ -535,6 +516,21 @@ def test_add_endpoint_args_without_custom_endpoint(self):
 
         self.assertEqual(expected_args, actual_args)
 
+    def test_add_endpoint_args_with_tensorrtllm_engine_endpoint(self):
+        """
+        Test that _add_endpoint_args returns the correct arguments
+        when tensorrtllm_engine endpoint is set
+        """
+        self._config.endpoint.service_kind = "tensorrtllm_engine"
+
+        expected_args = ["--service-kind", "triton_c_api", "--streaming"]
+
+        actual_args = self._default_perf_analyzer_config._add_endpoint_args(
+            self._config
+        )
+
+        self.assertEqual(expected_args, actual_args)
+
```

Comment:
As mentioned [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2057039805), GAP now appends `tensorrtllm_engine`-related PA options in `_add_endpoint_args()` instead of the previous `_add_protocol_args()`, so this test was added.